.github/workflows/ci_on_scratch.yml

```diff
@@ -0,0 +1,29 @@
+name: Normal CI
+
+on:
+  push:
+
+jobs:
+  run-test:
+    name: Run Test on Scratch
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: Install yarn
+        working-directory: ./frontend
+        run: npm install -g yarn
```

Comment:
```suggestion
```

Reply:
I wonder if this can be removed.

Reply:
Yes you can. Actuary I added for the local environment as act docker. But finally it didn't work, so Let's delete